File: public/phase-2-20160620022203/C_PLUS_PLUS/elite-architect.cpp
// Name : Elite Architect
#include <iostream>
#include <string>
using namespace std;

int main(int argc, char const *argv[]) {
	// input:
	// n : the number of Buildings in the capital of Nippon
	// h :  number of stories in each building
	int index;
	int n;
	cin >> n;
	int h[10000];

	for(index=0;index<n;index++)
		cin >> h[index];


	// write your code here
	// store your results in `a`

	// output
	// Dummy Data
	int a[10000]={3,2,0,2,0};

	for(index=0;index<n;index++)
		cout << a[index] << " ";

}

File: public/phase-2-20160620022203/C_SHARP/elite-architect.cs
// Name : Elite Architect
using System;
namespace Application {
	class elite_architect {
		public static void Main(string[] args) {
		// input:
		// n : the number of Buildings in the capital of Nippon
		// h :  number of stories in each building
		int index;
		int n = Convert.ToInt32(Console.ReadLine());
		int[] h = new int[n];
		string[] h_elements = (Console.ReadLine()).Split(' ');
		for(index=0;index<n;index++)
			h[index] = Convert.ToInt32(h_elements[index]);


		// write your code here
		// store your results in `a`

		// output
		// Dummy Data
		int[] a = new int[] {3,2,0,2,0};

		Console.WriteLine(string.Join(" ", a));
		}	
	}
}